server/grpc_service.go

```diff
@@ -342,7 +342,7 @@ func (s *Server) RegionHeartbeat(stream pdpb.PD_RegionHeartbeatServer) error {
 		storeLabel := strconv.FormatUint(storeID, 10)
 
 		regionHeartbeatCounter.WithLabelValues(storeLabel, "report", "recv").Inc()
-		regionHeartbeatLatency.WithLabelValues(storeLabel).Observe(float64(time.Now().Second()) - float64(request.GetTimestamp()))
+		regionHeartbeatLatency.WithLabelValues(storeLabel).Observe(float64(time.Now().Nanosecond()/int(time.Millisecond)) - float64(request.GetTimestamp()))
```

Comment:
Nanosecond -> UnixNano

Reply:
why changing here?  Mostly, seconds is preferred.

Reply:
I think using second is too inaccurate, in most cases the delay will be jumping between 0 and 1.

Reply:
why? you can set 0.001 as the base unit.

Reply:
Mostly, we use 0.0005

Reply:
It is not the metrics we are updating but the accuracy of time that tikv sends to pd.